README.md

```diff
@@ -1,6 +1,6 @@
 # localtunnel
 
-[![Build Status](https://travis-ci.org/localtunnel/localtunnel.svg?branch=master)](https://travis-ci.org/localtunnel/localtunnel)
+[![Build Status](https://travis-ci.org/localtunnel/localtunnel.svg?branch=master)](https://travis-ci.org/localtunnel/localtunnel) [![Known Vulnerabilities](https://snyk.io/test/github/localtunnel/localtunnel/badge.svg)](https://snyk.io/test/github/localtunnel/localtunnel)
```

Comment:
Well, the real intent of the PR was to help keep this project free of vulnerable dependencies... hence the badge and the test, showing this isn't a one-off fix but rather an ongoing intention to stay vuln free.

That said, the badge is just an optional way to show the world you care, no functional necessity in it. I'll remove it.